healthcheck/handler.go

```diff
@@ -9,13 +9,6 @@ import (
 	"github.com/ONSdigital/log.go/log"
 )
 
-// A list of possible health statuses
-const (
-	StatusOK       = "OK"
-	StatusCritical = "CRITICAL"
-	StatusWarning  = "WARNING"
-)
-
 // Handler responds to an http request for the current health status
 func (hc HealthCheck) Handler(w http.ResponseWriter, req *http.Request) {
 	now := time.Now().UTC()
```

Comment:
When trying to use this handler I realised that the uptime is returned in nanoseconds, whilst the spec defines that it should be in milliseconds. 
I don't know if this change would be in the scope of this task, but it's a small change. I would suggest to use the Millisecond constat:
```
	hc.Uptime = now.Sub(hc.StartTime) / time.Millisecond
```

Reply:
Added this to the nil-pointers PR